clis/stats-aggregation-cli/test/commands/__snapshots__/daily.test.ts.snap

```diff
@@ -3,41 +3,189 @@
 exports[`Daily command > when aggregating daily stats > should aggregate all entities' data when no entity is provided 1`] = `
 [
   {
+    "category": "OTHER",
     "day": 2022-10-16T00:00:00.000Z,
+    "rollup": null,
+    "totalBlobSize": 4300n,
+    "totalBlobs": 4,
+    "totalUniqueBlobs": 3,
+  },
+  {
+    "category": "ROLLUP",
+    "day": 2022-10-16T00:00:00.000Z,
+    "rollup": null,
+    "totalBlobSize": 2100n,
+    "totalBlobs": 2,
+    "totalUniqueBlobs": 2,
+  },
+  {
+    "category": null,
+    "day": 2022-10-16T00:00:00.000Z,
+    "rollup": null,
     "totalBlobSize": 6400n,
     "totalBlobs": 6,
     "totalUniqueBlobs": 3,
   },
   {
+    "category": null,
+    "day": 2022-10-16T00:00:00.000Z,
+    "rollup": "OPTIMISM",
+    "totalBlobSize": 2100n,
+    "totalBlobs": 2,
+    "totalUniqueBlobs": 2,
+  },
+  {
+    "category": "ROLLUP",
     "day": 2023-05-10T00:00:00.000Z,
-    "totalBlobSize": 7500n,
+    "rollup": null,
+    "totalBlobSize": 393216n,
+    "totalBlobs": 3,
+    "totalUniqueBlobs": 3,
+  },
+  {
+    "category": null,
+    "day": 2023-05-10T00:00:00.000Z,
+    "rollup": null,
+    "totalBlobSize": 393216n,
+    "totalBlobs": 3,
+    "totalUniqueBlobs": 3,
+  },
+  {
+    "category": null,
+    "day": 2023-05-10T00:00:00.000Z,
+    "rollup": "BASE",
+    "totalBlobSize": 393216n,
+    "totalBlobs": 3,
+    "totalUniqueBlobs": 3,
+  },
+  {
+    "category": "OTHER",
+    "day": 2023-08-03T00:00:00.000Z,
+    "rollup": null,
+    "totalBlobSize": 6200n,
     "totalBlobs": 6,
-    "totalUniqueBlobs": 6,
+    "totalUniqueBlobs": 2,
   },
   {
+    "category": null,
     "day": 2023-08-03T00:00:00.000Z,
+    "rollup": null,
     "totalBlobSize": 6200n,
     "totalBlobs": 6,
     "totalUniqueBlobs": 2,
   },
   {
+    "category": "OTHER",
+    "day": 2023-08-20T00:00:00.000Z,
+    "rollup": null,
+    "totalBlobSize": 2400n,
+    "totalBlobs": 2,
+    "totalUniqueBlobs": 1,
+  },
+  {
+    "category": "ROLLUP",
     "day": 2023-08-20T00:00:00.000Z,
+    "rollup": null,
```

Comment:
> Rollup field shouldn't be null when category is "ROLLUP", right?

For rows in the stats table, this can occur because the row represents an aggregation that includes all rollups, not just a single one. That’s why the rollup value is null.

The same applies to rows where both category and rollup are null; these rows represent total aggregations that include all elements.